collector/src/rustc-fake.rs

```diff
@@ -18,6 +18,11 @@ fn determinism_env(cmd: &mut Command) {
     cmd.env("RUSTC_FORCE_RUSTC_VERSION", "rustc-perf");
 }
 
+fn run_with_determinism_env(mut cmd: Command) {
+    determinism_env(&mut cmd);
+    assert!(cmd.status().expect("failed to spawn").success());
```

Comment:
Can we add a message here so that if the assert fails we get a good error message?